pkg/vf/virtio.go

```diff
@@ -328,6 +340,7 @@ func (dev *USBMassStorage) AddToVirtualMachineConfig(vmConfig *VirtualMachineCon
 	return nil
 }
 
+// Move these to top of file?
```

Comment:
I've removed this comment, but yeah it would make sense to group all hte public structs at the top of the file.